manifests/u/uGetdm/uGet/2.2.3/uGetdm.uGet.locale.en-US.yaml

```
@@ -1,23 +1,23 @@
-# Created using wingetcreate 0.4.1.1
-# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
-
-PackageIdentifier: uGetdm.uGet
-PackageVersion: 2.2.3
-PackageLocale: en-US
-Publisher: uGetdm.com
-PublisherUrl: https://ugetdm.com/
-PublisherSupportUrl: https://ugetdm.com/help/
-PackageName: uGet
-PackageUrl: https://ugetdm.com/
-License: LGPL-2.0
-ShortDescription: uGet is a lightweight yet powerful Open Source download manager for Windows.
-Moniker: uget
-Tags:
-- downloader
-- download-manager
-- foss
-- open-source
-- cross-platform
-ManifestType: defaultLocale
-ManifestVersion: 1.0.0
-
+# Created using wingetcreate 0.4.1.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: uGetdm.uGet
+PackageVersion: 2.2.3
+PackageLocale: en-US
+Publisher: uGetdm.com
+PublisherUrl: https://ugetdm.com/
+PublisherSupportUrl: https://ugetdm.com/help/
+PackageName: uGet
+PackageUrl: https://ugetdm.com/
```

```suggestion
#PublisherUrl: https://ugetdm.com/
#PublisherSupportUrl: https://ugetdm.com/help/
PackageName: uGet
#PackageUrl: https://ugetdm.com/
```
This server could not prove that it is ugetdm.com; its security certificate expired 219 days ago. This may be caused by a misconfiguration or an attacker intercepting your connection. Your computer's clock is currently set to Sunday, August 28, 2022. Does that look right? If not, you should correct your system's clock and then refresh this page.